src/tests/nurse/test_views.py

```diff
@@ -27,7 +31,21 @@ def patch_notify():
 @pytest.fixture
 def user(db):
     """Creates and yields a new user."""
-    user = User.objects.create(username=USERNAME)
+    user = User.objects.create(
+        username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME
+    )
+    user.set_password(PASSWORD)
+    user.save()
+    yield user
+    user.delete()
+
+
+@pytest.fixture
+def user1(db):
+    """Creates and yields a new user."""
+    user = User.objects.create(
+        username="username2", first_name=FIRSTNAME, last_name=LASTNAME
```

Comment:
why don't we name it `user2` if it's username2?

Reply:
fixed